azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/springcloud/AbstractMojoBase.java

```diff
@@ -267,6 +272,12 @@ public SpringCloudAppConfig getConfiguration() {
     public AppPlatformManager getAppPlatformManager() {
         if (this.manager == null) {
             final LogLevel logLevel = getLog().isDebugEnabled() ? LogLevel.BODY_AND_HEADERS : LogLevel.NONE;
+            final AzureEnvironment env = azureCredentialWrapper.getEnv();
+            final String environmentName = AuthHelper.azureEnvironmentToString(env);
+            if (env != AzureEnvironment.AZURE) {
```

Comment:
AzureEnvironment is enum ?
I prefer to user !AzureEnvironment.AZURE.equals(env), no matter what type it it.

Reply:
it is a standard constant field, like enum, there are no reasons to create another 'AZURE' enviromnent.